notebook2/notebook/notebook.py

```diff
@@ -106,6 +107,24 @@ def get_name(line):
         "containing the name of the docker image to use for worker pods.") from e
 
 
+def requires_auth(for_page = True):
+    def auth(f):
+        @wraps(f)
+        def decorated(*args, **kwargs):
+            if 'user' not in session:
+                # Redirect to Login page here
+                if for_page is True:
```

Comment:
`is True` on a boolean parameter feels wrong. I like `x is None` because it doesn't pun on the fact that `[]` or `""` are false-y. But here, we're not working with lists or strings, this is just a boolean parameter. Callees who call this function with a non-boolean parameter are clearly in the wrong and I don't much care if they get surprising behavior.